### PR TITLE
Update Java tutorial to add Gradle instructions

### PR DESCRIPTION
**Pull request by tchernobog**
Also add a mention to the vintage engine for JUnit 5 integration.

Also some minor updates to the new import names.

### CODE REVIEW

File: content/docs/guides/10-minute-tutorial.md [hunk 1]
```diff
@@ -116,6 +123,65 @@ Open the project in IntelliJ IDEA:
...
+apply plugin: 'java'
+sourceCompatibility = 11
+targetCompatibility = 11
+
+group = "hellocucumber"
+version = "1.0"
+
+// Versioning of dependencies
+wrapper.gradleVersion = '5.5.1'
+def cucumberVersion = '4.6.0'
```
Comment (mpkorstanje) [inline content/docs/guides/10-minute-tutorial.md]: Please use the Hugo version tags so this version is made dynamic

https://github.com/cucumber/docs.cucumber.io/blob/fc79bdf0fe30968d2cd2f44c43b410d589fe0ae7/content/docs/installation/java.md#L24

**Review by mpkorstanje — CHANGES_REQUESTED** [review]
Looks good.

**Comment by tchernobog** [comment]
Changes performed, thanks!

**Review by mpkorstanje — APPROVED** [review]

**Comment by aslakhellesoy** [comment]
Hi @tchernobog,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**